homeassistant/components/bittrex/sensor.py

```diff
@@ -63,9 +72,69 @@ def device_state_attributes(self):
         """Return additional sensor state attributes."""
         return {
             "symbol": self._symbol,
-            "last_trade_rate": self.coordinator.data[self._symbol]["lastTradeRate"],
-            "bid_rate": self.coordinator.data[self._symbol]["bidRate"],
-            "ask_rate": self.coordinator.data[self._symbol]["askRate"],
+            "last_trade_rate": self.coordinator.data[CONF_TICKERS][self._symbol][
```

Comment:
You could improve readability with a helper function to get `self.coordinator.data[CONF_TICKERS][self._symbol]`. The auto formatter's max line length handling in particular didn't do any favors to this line!

Reply:
That's a great idea - done! (Y)